style.css

```diff
@@ -0,0 +1,170 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  position: relative;
+}
+
+.body {
+  width: 100vw;
+  min-height: 100vh;
+  position: relative;
+}
+
+header {
+  position: sticky;
+  top: 0;
+}
+
+nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 30px;
+  background-color: #3c3a39;
+}
+
+.logo,
+.menu {
+  height: 24px;
+  cursor: pointer;
+}
+
+.flex {
+  display: flex;
+}
+
+.header_content {
+  padding: 0 8%;
+  padding-top: 100px;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: #1c1a19;
+  width: 100%;
+  height: 100vh;
+  background-image: url('/images/illustration-header-mobile.png');
+  background-repeat: no-repeat;
+  background-size: 50%;
+  background-position-x: right;
+  background-position-y: 60%;
+}
+
+.header_content h1 {
+  font-family: 'Crete Round', serif;
+  color: #ff6b00;
+  font-weight: 700;
+  font-size: 40px;
```

Comment:
Please stick to the design and fix the font-size and weight according to the Figma template.
<img width="1042" alt="Screenshot 2023-01-16 at 11 29 30 PM" src="https://user-images.githubusercontent.com/25887253/212761521-5ae26f93-aa6c-4d62-9cb2-1147c2c29455.png">
<img width="1042" alt="Screenshot 2023-01-16 at 11 29 56 PM" src="https://user-images.githubusercontent.com/25887253/212761522-c3cb3e8f-66af-4831-bfde-851e53c59a03.png">